packages/notifier/test/test-publish-kit.js

```diff
@@ -265,51 +264,8 @@ test('makeEmptyPublishKit - immediate fail', async t => {
   );
 });
 
-test('makePublishKit', async t => {
-  const expectedInitialCount = (
-    await makePublishKit(undefined).subscriber.subscribeAfter()
-  ).publishCount;
-  t.true(expectedInitialCount > 0n);
-  const initialValues = {
```

Comment:
Removing this eliminates coverage that arbitrary values can be published, but that coverage probably is of little value.